src/components/AudioPlayer/Player.js

```diff
@@ -87,7 +89,10 @@ const styles = {
     top: '50%',
     left: 0,
     marginTop: -15,
-    textAlign: 'center'
+    textAlign: 'center',
+    '& svg': {
+      verticalAlign: 'middle'
```

Comment:
Should be solved with an `IconContext`. In the frontend we already have one in [`_app.js`](https://github.com/orbiting/republik-frontend/blob/97ab3e4cbad1ac4aab4c533fa16a3a83344641e8/pages/_app.js#L61-L63):
```
<IconContext.Provider
  value={{ style: { verticalAlign: 'middle' } }}
>
```

We should add one here in the styleguide in `src/index.js` around everything. And re-export the context in `Icons/index.js` for usage in frontends. And mention the necessity of it in the icon docs.

Reply:
The [`@react-icons/all-files`](https://www.npmjs.com/package/@react-icons/all-files#configuration) have an import from the regular package for the context:
```
import { IconContext } from "react-icons";
```

But I'd would assume that importing from `@react-icons/all-files` also works. Since contexts are singleton it important to import from the same package everywhere.

Reply:
got it from @react-icons/all-files, seems to work fine